# J_Cell_Physiol
Bioinformatic identification of key pathways, hub genes, and microbiota for therapeutic intervention in Helicobacter pylori infection

Abstract
The pathogenic mechanisms of Helicobacter pylori infection remain to be defined, and potential interventional microbiota are just beginning to be identified. In this study, gene-set enrichment analysis (GSEA) was used to integrate three H. pylori infection microarray data sets from the gene expression omnibus database and identified ten hallmark gene sets and 35 Kyoto encyclopedia of genes and genomes (KEGG) pathways that differed between healthy and Helicobacter pylori-infected individuals. Weighted gene co-expression network analysis (WGCNA) performed on two of the data sets identified three key gene coexpression modules. These modules contained 54 enriched KEGG pathways, 25 of which overlapped with the GSEA analysis, suggesting potentially important roles in H. pylori-infection. We selected 116 hub genes from the three key modules for in vitro validation at the transcriptional level using H. pylori Sydney Strain 1 and verified the upregulation of 80. WGCNA of the microbiomes based on 20 mucosal samples and a sequence read archive data set revealed four microbiota modules correlated with H. pylori infection. The negatively correlated modules contained 11 microbiome families. These findings provide new insight into the pathogenesis of H. pylori infection and systematically identify 25 key pathways, 80 upregulated hub genes, and 11 families of candidate interventional microbiota for further research.

Keywords: GSEA; Helicobacter pylori; WGCNA; hub genes; microbiome.

https://pubmed.ncbi.nlm.nih.gov/32710499/
